### ISSUE TITLE
Remove U-WB option

### ISSUE THREAD

**Issue by nicholst** [NISOx-BDI/SwE-toolbox#26]
There are several options that @BryanGuillaume implemented as part of his thesis evaluations but were never found to be favorable

One example:  U-WB for the type of WB test: As noted in the help (`swe_cfg_design.m`)

>  The U-WB option is currently available, but nothing indicates that  it should be used and thus might be removed later.

WB small sample adjustments for resampling:
Lots of options, but they do correspond to what lots of people do... maybe keep?

... will update as I find more.

**Comment by nicholst** [comment]
Testing this now... ran into a few problems.  Not sure if they're specific to this PR, but they need to be addressed

I created a WB run with the Henson fMRI data (3 HRF basis coefficients, 12 subjects, 36 total scans).  SwEconfig (batch file save): [`SwE_config.mat`](https://www.dropbox.com/s/tts0pb5fjap284s/SwE_config.mat?dl=0).  I set a results directory *but* didn't take care to ensure that my PWD was the same as set in `swe.design.dir{1}`.  You know what happened?  When I executed the batch job started by "Specify Model" (as saved in `SwE_config.mat`), the [`SwE.mat`](https://www.dropbox.com/s/3jf0y8g0hcw2c5a/SwE.mat?dl=0) got created in the correct directory, but the output of "Run model" went to the current working directory!

It occurs to me while SPM's `SPM` structure has a `swd` at the top level, is it really the case that SwE's `SwE` doesn't have a corresponding `swd`?  Mine didn't.

**Comment by nicholst** [comment]
Next issue: 
In  `swe.design.WB.WB_yes.WB_nB` is 999 (the default), but when I did "Run model" it only got as far as 99:
```
Bootstrap # 96  Block 1/1               :  Elapsed time is 1.539202 seconds.
Bootstrap # 97  Block 1/1               :  Elapsed time is 1.550461 seconds.
Bootstrap # 98  Block 1/1               :  Elapsed time is 1.614926 seconds.
Bootstrap # 99  Block 1/1               :  Elapsed time is 1.617849 seconds.
                       ...done
Completed                               :          11:39:13 - 07/07/2018
...use the saved images for assessment
```
Also, notably, the 'thermometer bar' went to the top but then never reset... it's still got the spinning ball as I write.  However, I *was* then able to continue and do "Results".

**Comment by nicholst** [comment]
Next issue:  In results, the table looks great.  As far as I can tell it is correct (really would benefit from footer that tells you the number of Bootstraps, but that's a further enhancement).

One minor issue: There must be a missing semicolon somewhere b/c the following is spit out to the command line:

```
ans = 

  UIControl (hReg) with properties:

              Style: 'frame'
             String: ''
    BackgroundColor: [0.5800 0.7700 0.5700]
           Callback: ''
              Value: 0
           Position: [1.2111 1.2111 484.4444 230.1111]
              Units: 'pixels'

  Use GET to show all properties
```

**Comment by TomMaullin** [comment]
This was fixed in PR #32
